app/assets/stylesheets/embed.scss

```diff
@@ -153,7 +153,7 @@ footer {
   }
 }
 
-.logo {
+.aspect-image {
```

Comment:
The embed stylesheet is a completely other part of the app, this change shouldn't affect it at all. So we shouldn't modify this stylesheet.